extension/chrome/settings/modules/backup.ts

```diff
@@ -95,9 +101,9 @@ export class BackupView extends View {
     $(`#${showBlockName}`).css('display', 'block');
   }
 
-  public setHandlers = () => {
+  public setHandlers = async () => {
```

Comment:
making this function `async` means that `View.setTestViewStateLoaded();` may be called prior to its completion.
I modified `extension/js/common/view.ts` to await its completion in https://github.com/FlowCrypt/flowcrypt-browser/pull/3973/commits/9b1aa9fa55b51c43b2f1d79617b9bb3e1b6cab54, so should be good now.
Wasn't it appropriate to have this logic in `render()`? -- that's the only `async` method.

Reply:
>Wasn't it appropriate to have this logic in render()? -- that's the only async method.

You're right sir. It's really ugly to put some async function there. I just move those related functions to `backup.ts` itself. Thank you for your patience, sir!